### ISSUE TITLE
Aggregators with DataType as VARBINARY can not be created

### ISSUE THREAD

**Issue by anoopsjohn** [forcedotcom/phoenix#274]
When I try creating a new Aggregator of DataType VARBINARY and running, I am getting NPE

Caused by: java.lang.NullPointerException
    at com.salesforce.phoenix.expression.aggregator.DistinctCountAggregator.getSize(DistinctCountAggregator.java:131)
    at com.salesforce.phoenix.expression.aggregator.Aggregators.calculateSize(Aggregators.java:89)
    at com.salesforce.phoenix.expression.aggregator.Aggregators.<init>(Aggregators.java:61)
    at com.salesforce.phoenix.expression.aggregator.ServerAggregators.<init>(ServerAggregators.java:54)
    at com.salesforce.phoenix.expression.aggregator.ServerAggregators.deserialize(ServerAggregators.java:132)
    at com.salesforce.phoenix.coprocessor.UngroupedAggregateRegionObserver.doPostScannerOpen(UngroupedAggregateRegionObserver.java:155)
    at com.salesforce.phoenix.coprocessor.BaseScannerRegionObserver.postScannerOpen(BaseScannerRegionObserver.java:59)

Basically in PDataType we return null from getByteSize() of VARBINARY which causes this exception. For VARBINARY  we can not have a fixed width and isFixedWidth() return false..   Either return 0 or some other value from getByteSize() of VARBINARY rather than null or wherever we make use of getByteSize() check for isFixedWidth()
